src/components/Send.jsx

```diff
@@ -1,20 +1,168 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import { useState } from 'react'
 import { useLocation } from 'react-router-dom'
 import * as rb from 'react-bootstrap'
-import { serialize, ACCOUNTS } from '../utils'
+import PageTitle from './PageTitle'
+import ToggleSwitch from './ToggleSwitch'
+import { useCurrentWalletInfo, useSetCurrentWalletInfo, useCurrentWallet } from '../context/WalletContext'
+import { useSettings } from '../context/SettingsContext'
 import * as Api from '../libs/JmWalletApi'
 
-export default function Payment({ currentWallet }) {
+// not cryptographically random
+const pseudoRandomNumber = (min, max) => {
+  return Math.round(Math.random() * (max - min)) + min
+}
+
+const isValidAddress = (candidate) => {
+  return typeof candidate === 'string' && !(candidate === '')
+}
```

Comment:
Just FYI, some parts of a lib I've worked on in the past might be of interest if we ever want to do real address validation. See [this](https://github.com/swan-bitcoin/xpub-tool/blob/0c2cd9f090665f8eb23c935fed28837488444e03/packages/xpub-lib/src/validation.js) and [that](https://github.com/swan-bitcoin/xpub-tool/blob/0c2cd9f090665f8eb23c935fed28837488444e03/packages/xpub-lib/src/validation.test.js#L125).